configuration/config2/XSDValueSetParser.hpp

```diff
@@ -0,0 +1,56 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2015 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+
+#ifndef _CONFIG2_XSDVALUESETPARSER_HPP_
+#define _CONFIG2_XSDVALUESETPARSER_HPP_
+
+#include <string>
+#include <memory>
+#include <map>
+#include <boost/property_tree/ptree.hpp>
+
+#include "XSDConfigParser.hpp"
+#include "ConfigItemValueSet.hpp"
+
+namespace pt = boost::property_tree;
```

Comment:
was there a reason to choose boost::property_tree over a jlib equivalent?

Reply:
Much of it was because of prior knowledge of how to use boost property trees. Looking at the jlib trees didn't show much in the way of documentation and looking at code examples showed that it would have taken longer to get the code up and running. We can discuss later if you like.

Reply:
The thing to ask ourselves is whether this is going to make others reluctant to make any changes to this code as everyone is more familiar with JLIB.